vllm/model_executor/layers/quantization/fp8_rocm.py

```diff
@@ -0,0 +1,299 @@
+from typing import Any, Dict, List, Optional, Tuple, Union, Iterator
+
+import torch
+from torch.nn import Module
+from torch.nn.parameter import Parameter
+import torch.nn.functional as F
+from safetensors import safe_open
+
+from vllm import _custom_ops as ops
+from vllm.logger import init_logger
+from vllm.model_executor.layers.linear import LinearBase, LinearMethodBase
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig)
+from vllm.model_executor.utils import set_weight_attrs
+
+from vllm._C import ops
```

Comment:
You should handle the Pybind methods inside `_custom_ops.py`:
https://github.com/ROCm/vllm/blob/fp8-gemm/vllm/_custom_ops.py